src/functionalTest/resources/features/F-011/F-011.feature

```diff
@@ -0,0 +1,34 @@
+@F-011
+Feature: F-011 : Verify CRD and JRD service bus messages
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-211
+  @FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on)
```

Comment:
Also needs the same feature toggle as `F-010/@S-101` which it was coppied from.

```
@FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on) @FeatureToggle(DB:employment_wa_1_0=on)
```

Reply:
Updated with @FeatureToggle(DB:employment_wa_1_0=on)

Reply:
Resolved in 4e0f7f55cae6678e4005fd17ca6d9e24538e691d